app/code/Magento/Customer/Model/Customer/DataProvider.php

```diff
@@ -289,7 +289,7 @@ private function overrideFileUploaderData($entity, array &$entityData)
      */
     private function getFileUploaderData(
         Type $entityType,
-        Attribute $attribute,
+        \Magento\Eav\Model\Entity\Attribute $attribute,
```

Comment:
According to use block, this is not a `Magento\Eav\Model\Entity\Attribute`, it is `Magento\Customer\Model\Attribute` the parameter type is correct and does not need to be changed.